components/scheduler/src/Scheduler.svelte

```diff
@@ -250,8 +250,8 @@
       event,
     });
   }
-  function unhoverOption(event: ConsecutiveEvent[]) {
-    dispatchEvent("eventOptionUnhovered", {
+  function blurOption() {
+    dispatchEvent("eventOptionHovered", {
```

Comment:
Unhovered becoming Hovered -> a typo?

Reply:
Nope, I just removed `unhovered` as a separate event. Now we just emit `hovered` with value null to reduce the number of event handlers needed.

Reply:
Hmmmm, in that case renaming to onHoverOptionChange seems more complete.